packages/sdk/src/3.1/models.ts

```diff
@@ -2477,14 +2477,6 @@ export enum Format {
   vector_tile_region = 'vector_tile_region',
 }
 
-/**
- * Scheme that is running on application server (for PRs, file browsing, etc.) Valid values are: "http", "https".
- */
-export enum GitApplicationServerHttpScheme {
-  http = 'http',
-  https = 'https',
-}
-
```

Comment:
@jkaster this feels like borderline breaking for 3.1 - what do you think?

Reply:
This enum was removed because it was not accurate. Sometimes `null` or an empty string would be provided instead, so it broke the Kotlin SDK for example. It was wrong for both 3.1 and 4.0, so I'd consider it a bug fix?